HangmanGame/ViewController.swift

```diff
@@ -8,12 +8,165 @@
+    @IBOutlet weak var textBox: UITextField!
+    
+    var score = 0 {
+        didSet {
+            lblScore.text = "Score \(score)"
+        }
+    }
+    
+    var wrongLetters = [String]()
+    var questionMark = [String]()
+    var words = ["MOOSEUM","ROADHOG","TRUNK","MILKSHAKE","BABOOM","BULLDOZER","HISSTORY"]
+    var selectedWord = ""
+    var letters = [Any]()
+    var numberOfLetters = 0
+    var wrongTempLetter = ""
+    var rightTempLetter = ""
+    var counter = 0
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        words.shuffle()
+        loadLevel()
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .refresh, target: self, action: #selector(loadLevel))
+        navigationController?.isToolbarHidden = false
+        
+    }
+    
+    private func divideLetters() {
+        selectedWord = words[1]
+        numberOfLetters = selectedWord.count
+        letters = Array(selectedWord)
+        questionMark.removeAll()
+        correctWord.text = ""
+        for index in 0...numberOfLetters-1{
+            questionMark.append("?")
+            correctWord.text! += questionMark[index]
+        }
+    }
+    
+    private func hideImages() {
+        imgHead.isHidden = true
+        imgBody.isHidden = true
+        imgLeftArm.isHidden = true
+        imgRightArm.isHidden = true
+        imgLeftLeg.isHidden = true
+        imgRightLeg.isHidden = true
+    }
+    
+    private func clearFields() {
+        lblWrongGuess.text = ""
+    }
+    
+    private func showImages() {
+        switch(counter) {
+        case 1:
+            imgHead.isHidden = false
+            break
+        case 2:
+            imgBody.isHidden = false
+            break
+        case 3:
+            imgLeftArm.isHidden = false
+            break
+        case 4:
+            imgRightArm.isHidden = false
+            break
+        case 5:
+            imgLeftLeg.isHidden = false
+            break
+        case 6:
+            imgRightLeg.isHidden = false
+            gameOver()
+            break
+        default:
+            break
+        }
+    }
+    
+    private func alertMsg(title: String, message: String) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+        self.present(alert, animated: true, completion: nil)
+    }
+    
+    private func gameOver() {
+        score -= 1
+        alertMsg(title: "Game Over", message: "You have lost 1 point")
+        loadLevel()
+    }
+    
+    @objc private func loadLevel() {
+        clearFields()
+        hideImages()
+        words.shuffle()
+        divideLetters()
+    }
+    private func victory() {
+        alertMsg(title: "Victory", message: "You've got 1 point")
+        score += 1
+    }
+    
+    @IBAction func btnTryClicked(_ sender: UIButton) {
```

Comment:
Just so that we can better review your code please put in the some description of the requirement either in the PR, project README or when it makes sense on the function documentation.

Reply:
I'll update the requirements!